CVEs/CVE-2020-26256.json

```
@@ -0,0 +1,24 @@
+{
+    "CVE": "CVE-2020-26256",
+    "state": "PUBLISHED",
+    "repository": "https://github.com/C2FO/fast-csv.git",
+    "prePatch": {
+        "commit": "b03b546f2ba55761bef96e43f78801fa342436d3",
```

```suggestion
        "commit": "1d18b894236687a5865ea70647010c1697a61d6d",
```
Changed to the `prePatch` commit to be `postPatch^`, the weakness location remains the same.